### DISCUSSION TITLE
有无办法通过SoraApi示例判断api链接是否有效

### DISCUSSION THREAD

**Discussion by cc004** [Hoshikawa-Kaguya/Sora#51] [Q&A]
rt

**Comment by Yukari316** [comment]
可以试试
```csharp
SoraApi api;
(ApiStatus staus, bool online, bool good, JObject stat) = await api.GetStatus();
```
一般如果gocq服务链接挂掉的话会报错`time out`
![image](https://user-images.githubusercontent.com/7535224/150838512-43cc4f6c-10ec-40bd-94c0-82def7d5049f.png)
